lib/bolt/cli.rb

```diff
@@ -564,19 +564,121 @@ def generate_types
       0
     end
 
+    # Initializes a specified directory as a Bolt project and installs any modules
+    # specified by the user, along with their dependencies
     def initialize_project
-      path = File.expand_path(options[:object] || Dir.pwd)
-      FileUtils.mkdir_p(path)
-      ok = FileUtils.touch(File.join(path, 'bolt.yaml'))
+      boltdir    = Pathname.new(File.expand_path(options[:object] || Dir.pwd))
+      config     = boltdir + 'bolt.yaml'
+      puppetfile = boltdir + 'Puppetfile'
+      modulepath = [boltdir + 'modules']
+      plural     = options[:modules]&.count == 1 ? '' : 's'
```

Comment:
I love to rephrase messages to avoid the need for pluralization. Is that possible in this case?

Reply:
Removing most of the plurals. The only one I'm not sure how to reword is `Unknown module(s) ...`...